src/main/java/com/gooddata/visualizationObject/VisualizationType.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2007-2017, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+
+package com.gooddata.visualizationObject;
+
+public enum VisualizationType {
+    TABLE,
+    LINE,
+    COLUMN,
+    BAR,
+    PIE,
+
+    // fallback to prevent 500
```

Comment:
I don't get it - if the res validation is performed, it should throw 400

Reply:
actually every unknown type is converted to table, so UNKNOWN type, and probably get as well, is unnecessary